tests/test_response_handler.py

```diff
@@ -2,7 +2,12 @@
 
 from cornice_swagger.swagger import ResponseHandler, CorniceSwaggerException
 from cornice_swagger.converters import convert_schema
-from .support import BodySchema, HeaderSchema, ResponseSchema, response_schemas
+from .support import (BodySchema,
```

Comment:
Okay, sure. I'm used to the pylons project code style :) I will fix in the next commit.